COLLABORATOR_GUIDE.md

```diff
@@ -56,16 +56,23 @@ full responsibility for the change, defer to another Collaborator.
 Before landing pull requests, sufficient time should be left for input
 from other Collaborators. Leave at least 48 hours during the week and
 72 hours over weekends to account for international time differences
-and work schedules. Trivial changes (e.g. those which fix minor bugs
-or improve performance without affecting API or causing other
-wide-reaching impact) may be landed after a shorter delay.
+and work schedules. Trivial changes (e.g. fixing typographical errors in
+documentation) may be landed after a shorter delay.
```

Comment:
I changed it because I thought it was outdated and the change I made here would be a non-controversial representation of current thinking/practice. Looks like I was wrong, so I'll undo that and we can hopefully move any conversation about it and the 48/72 hour rules to some other issue/venue. I guess this is an example of what @bnoordhuis is talking about with piling little unrelated changes into a PR just to get them through. :-(